static/app/components/replays/replayContext.tsx

```diff
@@ -61,13 +63,21 @@ export function Provider({children, events}: React.PropsWithChildren<Props>) {
   const [isPlaying, setIsPlaying] = useState(false);
   const [skipInactive, setSkipInactive] = useState(false);
   const [speed, setSpeedState] = useState(1);
+  const [fastForwardSpeed, setFFSpeed] = useState(0);
 
   const forceDimensions = dimension => {
     setDimensions(dimension as Dimensions);
   };
   const setPlayingFalse = () => {
     setIsPlaying(false);
   };
+  const onFastForwardStart = (e: unknown) => {
+    // @ts-expect-error: rrweb has poor types for event handlers
```

Comment:
What is the error exactly?

Reply:
The error is that all the event handlers are typed as `(e: unknown) => void` so there isn't any inference to match teh event type with its payload. 

I ended up writing the handler this way: 
```
const onFastForwardStart = e => {
  setFFSpeed((e as {speed: number}).speed);
};

inst.on(ReplayerEvents.SkipStart, onFastForwardStart);
```

Reply:
If you type it like this is it unhappy?

```tsx
const onFastForwardStart = (e: {speed: number}) => {
  setFFSpeed(e.speed);
};
```

Reply:
yes it is unhappy :(

The error moves down to the `.on()` call:
<img width="1214" alt="Screen Shot 2022-04-18 at 1 53 59 PM" src="https://user-images.githubusercontent.com/187460/163876372-2af234ab-db77-4440-9c4c-3b576ae21ba6.png">

And the `Handler` type that it mentions is the issue:
<img width="550" alt="Screen Shot 2022-04-18 at 1 53 33 PM" src="https://user-images.githubusercontent.com/187460/163876422-f70b1fa0-759f-46a7-8459-f217a8a8bfed.png">

Reply:
lol so annoying. IMO I would keep the ts-expect-error in that case

``tsx
// @ts-expect-error: rrweb types event handlers with `unknown` parameters
inst.on(ReplayerEvents.SkipStart, onFastForwardStart);
```

Reply:
updated